client/rest/src/plugins/rosetta/constructionRoutes.js

```diff
@@ -31,6 +35,8 @@ export default {
 		const networkName = services.config.network.name;
 		const network = NetworkLocator.findByName(Network.NETWORKS, networkName);
 
+		const restUrl = `${services.config.rest.protocol}://localhost:${services.config.rest.port}`;
+
```

Comment:
certs(https) are not created for localhost.  I will need to see how this works with a reverse proxy.